packages/strapi-plugin-content-type-builder/admin/src/translations/fr.json

```diff
@@ -89,6 +89,9 @@
 
   "modelPage.attribute.relationWith": "Relation avec",
 
+  "noTableWarning.description": "N'oubliez pas de créer `{modelName}` dans votre database",
+  "noTableWarning.infos": "Plus d'infos",
```

Comment:
Did you create the documentation section for this kind of message?

Reply:
Not yet it will be on another branch where I need to fix some mistakes in the doc